model/host/host_test.go

```diff
@@ -324,6 +324,43 @@ func TestHostSetDNSName(t *testing.T) {
 	})
 }
 
+func TestHostSetIPv6Address(t *testing.T) {
+	var err error
+
+	Convey("With a host", t, func() {
```

Comment:
For new tests, you should use the testify library (look for any tests that start with assert := assert.New(t))